change/@microsoft-teams-js-d5fe13cd-d446-4761-8abe-f733ed6aa57b.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "extract timeout logic into a func & add test",
```

Comment:
>extract timeout logic into a func & add test [](http://example.com/codeflow?start=14&length=44)

Since these notes appear in the public changelog, can you provide a little more detail about the change so that developers can better understand the change? Also please use past-tense to ensure consistency. Suggestion:

Added timeout notifications (2 seconds) to video frame processing in `videoEffectsEx` capability